.kokoro/build.sh

```diff
@@ -71,8 +71,29 @@ integration)
     ;;
 graalvm)
     # Run Unit and Integration Tests with Native Image
-    mvn -B ${INTEGRATION_TEST_ARGS} -ntp -Pnative -Penable-integration-tests test
-    RETURN_CODE=$?
+#    if [[ "${KOKORO_GITHUB_PULL_REQUEST_NUMBER}" = "" ]]
+#    then
+#      mvn -B ${INTEGRATION_TEST_ARGS} -ntp -Pnative -Penable-integration-tests test
+#      RETURN_CODE=$?
+#    branchName=$(git name-rev 9188d10f710ce97b884dc8c805002ded08144e70)
+#    BRANCH="$(git branch --no-color --contains "$(git rev-parse HEAD)" | grep -v 'HEAD detached' || exit 0)"
+#    BRANCH="${BRANCH/  /}"
+#    readonly BRANCH
+#    case "${BRANCH:-}" in
+#    "check-kokoro")
+#      ;;
+#    *)
+#      # For now, only generate documentation on pushes to `master`.
+#      echo "The branch name is NOT check-kokoro."
+#      exit 0
+#      ;;
+#    esac
+#
+    CURRENT_BRANCH=$(git branch | grep \* | cut -d ' ' -f2)
+    echo "Current branch: ${CURRENT_BRANCH}"
```

Comment:
Can you add "printenv" command to print all environment variables? There might be a variable that holds what you're looking for.

Reply:
great idea! Just tried this out and unfortunately there doesn't seem to be a variable for the kokoro branch name but doing these two commands looks promising:

```
branchName=$(git name-rev 9188d10f710ce97b884dc8c805002ded08144e70)
echo $branchName
```

Result: `9188d10f710ce97b884dc8c805002ded08144e70 remotes/origin/check-kokoro~24`

The `KOKORO_GIT_COMMIT` could be useful.